README.md

```diff
@@ -126,19 +126,22 @@ If you want to redeploy to the same production URL you can run `yarn vercel --pr
 
 ## Interacting with your Smart Contracts: SE-2 Custom Hooks
 
-Scaffold-ETH 2 provides a collection of custom React hooks designed to simplify interactions with your deployed smart contracts. These hooks are wrappers around `wagmi`, automatically loading the necessary contract ABI and address. They offer an easy-to-use interface for reading from, writing to, and monitoring events emitted by your smart contracts.  
-If you need to interact with external contracts, you can use `wagmi` directly, or add external contract data to your `deployedContracts.ts` file.
+Scaffold-ETH 2 provides a collection of custom React hooks designed to simplify interactions with your deployed smart contracts. These hooks are wrappers around `wagmi`, automatically loading the necessary contract ABI and address. They offer an easy-to-use interface for reading from, writing to, and monitoring events emitted by your smart contracts.
 
-- [useScaffoldContractRead](#usescaffoldcontractread)
-- [useScaffoldContractWrite](#usescaffoldcontractwrite)
-- [useScaffoldEventSubscriber](#usescaffoldeventsubscriber)
-- [useScaffoldEventHistory](#usescaffoldeventhistory)
-- [useDeployedContractInfo](#usedeployedcontractinfo)
-- [useScaffoldContract](#usescaffoldcontract)
+To help developers get started with smart contract interaction using Scaffold-ETH 2, we've provided the following custom hooks:
+
+- [useScaffoldContractRead](#usescaffoldcontractread): for reading public variables and getting data from read-only functions of your contract.
```

Comment:
Not strong on adding descriptions here, like it seem repetitive to OG description(also a bit conservative about README length 😅) but yeah it makes sense to give quick one-liner about the hooks 

but would love to know other's thoughts

Reply:
I think the quick one-liner about the hooks makes sense and helps to know better what the hooks actually do before going to the detailed section.

Reply:
I wasn't sure either, seems like pretty self-explanatory, but also think the one liner can make it more explicit. On the readme and all the documentation I generally try to default to the more info the better.

Reply:
> I think the quick one-liner about the hooks makes sense and helps to know better what the hooks actually do before going to the detailed section.

> but also think the one-liner can make it more explicit.

Yup 💯💯